exercises/dnd-character/.meta/template.j2

```diff
@@ -0,0 +1,33 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{% set class = exercise | camel_case -%}
+{{ macros.header(["Character", "modifier"]) }}
+
+class {{ exercise | camel_case }}Test(unittest.TestCase):
+    {% for case in cases[0]["cases"] -%}
+    def test_{{ case["description"] | replace("-","n") | to_snake }}(self):
+        {% set value = case["input"]["score"] -%}
+        {% set expected = case["expected"] -%}
+        self.assertEqual({{ case["property"] }}({{ value }}), {{ expected }})
+
+    {% endfor %}
+
+    def test_random_ability_is_within_range(self):
+        score = Character().{{ cases[1]["property"] }}()
+        self.assertTrue({{ cases[1]["expected"] | replace("&&","and") }})
+
+    def test_random_character_is_valid(self):
+        Char = Character()
+        {% for ability in cases[2]["expected"] -%}
+        {% set statement = cases[2]["expected"][ability] | replace("&&","and") | replace(ability, ["Char.", ability ]|join("")) -%}
+        {% if ability == "hitpoints" -%}
+        {% set statement = statement | replace("constitution","Char.constitution") -%}
+        {%- endif -%}
+        self.assertTrue({{ statement }})
+        {% endfor %}
+
+    def test_each_ability_is_only_calculated_once(self):
+        Char = Character()
+        {% set ability = cases[3]["property"] -%}
+        self.assertTrue({{ cases[3]["expected"] | replace(ability , ["Char.", ability]|join(""))}})
```

Comment:
To avoid false-positives on [truthy](https://stackoverflow.com/q/39983695/3229611) values, please use `assertIs(_, True)` instead of `assertTrue` and `assertIs(_, False)` instead of `assertFalse`.

```suggestion
        self.assertIs({{ cases[3]["expected"] | replace(ability , ["Char.", ability]|join(""))}}, True)
```